### PR TITLE
fix: serializeNode was naively checking if values were falsey

### PR DESCRIPTION
**Pull request by dbousamra**
This fixes https://github.com/prevwong/craft.js/issues/110

I haven't added any tests, as I noticed there were already none for this function. I can add some if you want.

### CODE REVIEW

File: packages/core/src/utils/serializeNode.tsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ export const serializeComp = (
   let { type, isCanvas, props } = data;
   props = Object.keys(props).reduce((result: Record<string, any>, key) => {
     const prop = props[key];
-    if (!prop) {
+
+    if (prop == null) {
```
Comment (prevwong) [inline packages/core/src/utils/serializeNode.tsx:21]: Good catch! 💯 Can you use strict equality `===` instead?

**Review by prevwong — CHANGES_REQUESTED** [review]

File: packages/core/src/utils/serializeNode.tsx (hunk 1 above)
Comment (dbousamra) [reply packages/core/src/utils/serializeNode.tsx:21]: Sure. Is triple equals the right way here? == will check against undefined, not just null.

Comment (prevwong) [reply packages/core/src/utils/serializeNode.tsx:21]: I mean something a bit more verbose like `prop === undefined || prop === null`. I think this makes the code a little bit more readable and potentially less bug prone.

**Review by prevwong — APPROVED** [review]
Thanks!

**Pull request merged**